src/benchmarks/micro/Categories.cs

```diff
@@ -21,15 +21,20 @@ public static class Categories
         /// benchmarks belonging to this category are executed for CI jobs
         /// </summary>
         public const string Runtime = "Runtime";
-            public const string BenchmarksGame = "BenchmarksGame";
-            public const string Benchstones = "Benchstones";
-                public const string BenchF = "BenchF";
-                public const string BenchI = "BenchI";
-            public const string Inlining = "Inlining";
-            public const string V8 = "V8";
-            public const string Perflab = "Perflab";
-            public const string Virtual = "Virtual";
+        public const string BenchmarksGame = "BenchmarksGame";
+        public const string Benchstones = "Benchstones";
+        public const string BenchF = "BenchF";
+        public const string BenchI = "BenchI";
+        public const string Inlining = "Inlining";
+        public const string V8 = "V8";
+        public const string Perflab = "Perflab";
+        public const string Virtual = "Virtual";
+        public const string ByteMark = "ByteMark";
```

Comment:
the `ByteMark` category is used only by a single type. What is the purpose of adding it?